### ISSUE TITLE
Async publishing from non-Windows platforms always fails during artifact creation 

### ISSUE THREAD

**Issue by natemcmaster** [dotnet/arcade#3046]
I attempted to on-board AspNetCore to publishing with Arcade, but immediately ran into this error message on all non-Windows platforms.

![image](https://user-images.githubusercontent.com/2696087/59453041-e3b5ea80-8dc3-11e9-81ab-186201a1c4bb.png)
https://dev.azure.com/dnceng/internal/_build/results?buildId=223311&view=logs
```
##[error]Unable to process command '  ##vso[artifact.upload containerfolder=ReleaseConfigs;artifactname=ReleaseConfigs]/Users/vsts/agent/2.152.1/work/1/s/eng/common\PublishToPackageFeed.proj' successfully. Please reference documentation (http://go.microsoft.com/fwlink/?LinkId=817296)
##[error]Path not exist: /Users/vsts/agent/2.152.1/work/1/s/eng/common\PublishToPackageFeed.proj
##[error]Unable to process command '  ##vso[artifact.upload containerfolder=ReleaseConfigs;artifactname=ReleaseConfigs]/Users/vsts/agent/2.152.1/work/1/s/eng/common\PublishToSymbolServers.proj' successfully. Please reference documentation (http://go.microsoft.com/fwlink/?LinkId=817296)
##[error]Path not exist: /Users/vsts/agent/2.152.1/work/1/s/eng/common\PublishToSymbolServers.proj
##[error]Unable to process command '  ##vso[artifact.upload containerfolder=ReleaseConfigs;artifactname=ReleaseConfigs]/Users/vsts/agent/2.152.1/work/1/s/eng/common\CheckSymbols.ps1' successfully. Please reference documentation (http://go.microsoft.com/fwlink/?LinkId=817296)
##[error]Path not exist: /Users/vsts/agent/2.152.1/work/1/s/eng/common\CheckSymbols.ps1
```

My guess: this would be fixed by normalizing file paths to use forward slashes on Linux/macOS.

cc @JunTaoLuo @markwilkie

**Comment by markwilkie** [comment]
Yea...that looks silly.  I presume this blocks you guys from moving to arcade on aspnetcore?

cc/ @jcagme @riarenas @JohnTortugo @mmitche

**Comment by natemcmaster** [comment]
It doesn't block us from _building_ with arcade, just blocks us from _publishing_. We can still use the same pipeline that we've been using with KoreBuild.

**Comment by riarenas** [comment]
Yeah... silly indeed. https://github.com/dotnet/arcade/pull/3047 should fix it, and I created #3048 so that Arcade-Validation catches these in the future.
